tests/suites/test_suite_psa_crypto_metadata.function

```diff
@@ -735,3 +735,143 @@ void lifetime(int lifetime_arg, int classification_flags,
     TEST_EQUAL(PSA_KEY_LIFETIME_GET_LOCATION(lifetime), location);
 }
 /* END_CASE */
+
+/* BEGIN_CASE */
+void constant_sizes()
+{
+    /* The goal of this test function is to verify that size macros
+     * are compile-time constants. If they aren't, the build will fail
+     * if the compiler doesn't accept variable-length arrays. */
+    /* Define each argument name as a compile-time constant. This can be
+     * almost anything since we don't care about the value. If you add one,
+     * don't forget to #undef it below. */
+#define alg 0
+#define bits 0
+#define block_size 1 // 0 would cause a division by zero
+#define bytes 0
+#define ciphertext_length 0
+#define curve_bits 0
+#define input_length 0
+#define key_bits 0
+    /* We need something that looks like a block cipher to avoid a division
+     * by zero in
+     * PSA_ROUND_UP_TO_MULTIPLE(PSA_BLOCK_CIPHER_BLOCK_LENGTH(key_type), (input_length))
+     * which would be undefined behavior and, in practice, causes a
+     * compilation failure when building with `gcc -Werror=vla -fundefined`.
+     */
+#define key_type PSA_KEY_TYPE_AES
+#define length 0
+#define plaintext_length 0
+    union all_sizes_must_be_constant {
+        /* For every size macro: unsigned char a_FOO[1 + FOO(...)].
+         * Add 1 so that the result is guaranteed to be nonzero (since we
+         * don't bother to pass "sensible" parameters to the size macros,
+         * most of the calls return 0).
+         */
+        // The fields of this union were generated with the following command:
+        //     sed -n 's/^ *# *define  *\([A-Z_a-z][0-9A-Z_a-z]*\)\(([^()]*)\).*/        unsigned char a_\1[1 + \1\2];/p' include/psa/crypto_sizes.h
+        unsigned char a_PSA_BYTES_TO_BITS[1 + PSA_BYTES_TO_BITS(bytes)];
```

Comment:
Is it deliberate that `unsigned char a_PSA_BITS_TO_BYTES[1 + PSA_BITS_TO_BYTES(bits)];` is missing?

Reply:
No. I probably just accidentally left out the line when copying the output of the sed command.

Checking, there's one other discrepancy:
```
<         unsigned char a_PSA_KEY_EXPORT_ASN[1 + PSA_KEY_EXPORT_ASN1_INTEGER_MAX_SIZE(bits)];
---
>         unsigned char a_PSA_KEY_EXPORT_ASN1_INTEGER_MAX_SIZE[1 + PSA_KEY_EXPORT_ASN1_INTEGER_MAX_SIZE(bits)];
815,831d39
```
I have no idea how that one happened.